docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '2'
+
+services:
+   db:
+     image: sameersbn/mysql
+     volumes:
+       - db_data:/var/lib/mysql
+     restart: always
+     environment:
+       DB_NAME: wordpress
+       DB_USER: wordpress
+       DB_PASS: wordpress
+
+   wordpress:
+     depends_on:
+       - db
+     image: wordpress:latest
+     ports:
+       - "8000:80"
+     restart: always
```

Comment:
[It ensures that container is always restarted by the Docker daemon if for some reason it stops.](http://stackoverflow.com/questions/29603504/how-to-restart-an-existing-docker-container-in-restart-always-mode) I guess so that if everything else is running and the db stops, it gets restarted. Good for production, not sure how useful it is for dev